### DISCUSSION TITLE
Multiple x data alpinejs with livewire

### DISCUSSION THREAD

**Discussion by rizkhal** [livewire/livewire#2968] [1. Help] [answered]
I have `x-data` in my base layouts like below to make switch theme
![Screenshot from 2021-06-15 17-47-02](https://user-images.githubusercontent.com/24653114/122032286-10ad9580-ce02-11eb-82fa-efdabc4f511d.png)

and in my other file extending the base layout, i write `x-data` again to make reusable modal like below
![Screenshot from 2021-06-15 17-47-19](https://user-images.githubusercontent.com/24653114/122032431-333fae80-ce02-11eb-9b1b-eb3d363f27d5.png)

it works now, but i wonder if making it like this is correct?

Thanks for this amazing package ❤️

**Comment by joshhanley** [comment, marked as answer]
@rizkhal nothing wrong with that 🙂
